static/js/robot.js

```diff
@@ -677,6 +681,7 @@ class Simulator{
         this.robot = null;
         this.mode = "idle";
         this.initGamepad();
```

Comment:
Could you change this to `this.gamepad = new GamepadClass();` and remove the argument in the `GamepadClass` constructor?